src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -163,6 +166,9 @@
     private static final String OPTION_TREE_WALKER_THREADS_NUMBER_NAME =
         "tree-walker-threads-number";
 
+    /** Name for the option 'gxs'. */
+    private static final String OPTION_GXS_NAME = "gxs";
```

Comment:
name it "generate-xpath-suppression" and no short form is required. This mode should not be used very often.